src/components/TextInput/TextInput.mdx

```diff
@@ -163,8 +164,8 @@ the `type` prop is set to `"textarea"`.
 
 ### Controlled isClearableCallback Example
 
-A `TextInput` component is controlled when the `value` and `onChange` props are
-set. For a controlled component, the `isClearableCallback` prop can be used to
+A `TextInput` component is controlled when the `value` and `onChange` attributes are
+passed. For a controlled component, the `isClearableCallback` prop can be used to
 clear the input value in the consuming application. This callback is only called
 when the close button is clicked.
```

Comment:
I know this was already in here, but I don't think `close button` is appropriate. It looks like the close button used in other components, but that should not dictate how the button is interpreted. Unless I am misunderstanding why is was called the `close button`, let's update the text to say `clear button`.